homeassistant/components/lutron/__init__.py

```diff
@@ -4,13 +4,14 @@
 from pylutron import Button, Lutron
 import voluptuous as vol
 
+from homeassistant import config_entries
 from homeassistant.const import ATTR_ID, CONF_HOST, CONF_PASSWORD, CONF_USERNAME
-from homeassistant.helpers import discovery
+from homeassistant.helpers import area_registry as ar, device_registry as dr
```

Comment:
Please use longer variable names than two characters.

Reply:
@MartinHjelmare I'm asking to learn, but this [example](https://developers.home-assistant.io/docs/device_registry_index/#manual-registration) shows something very similar, so is the example incorrect?

Reply:
This particular case is fine. Normally we want at minimum three characters.